activesupport/lib/active_support/core_ext/class/attribute.rb

```diff
@@ -70,9 +70,10 @@ class Class
   # To opt out of both instance methods, pass <tt>instance_accessor: false</tt>.
   def class_attribute(*attrs)
     options = attrs.extract_options!
-    instance_reader = options.fetch(:instance_accessor, true) && options.fetch(:instance_reader, true)
-    instance_writer = options.fetch(:instance_accessor, true) && options.fetch(:instance_writer, true)
+    instance_reader    = options.fetch(:instance_accessor, true) && options.fetch(:instance_reader, true)
+    instance_writer    = options.fetch(:instance_accessor, true) && options.fetch(:instance_writer, true)
     instance_predicate = options.fetch(:instance_predicate, true)
+    default_value      = options.fetch(:default, nil)
```

Comment:
Would be nice to convert all these to kwargs. Should probably do a big sweep for Rails 6.